### DISCUSSION TITLE
Choosing a Logo

### DISCUSSION THREAD

**Discussion by xynydev** [blue-build/.github#4] [Polls]
With all current votes in [the name poll](https://github.com/orgs/blue-build/discussions/1) going to the _BlueBuild_ name, I'll now start the vote between the already made logos for the name.

### Option A – Inspired by the [blue-billed duck](https://duckduckgo.com/?q=blue-billed+duck&iax=images&ia=images)
Designed by Ian Price per @gmpinder's commission.
![BlueBuild-01](https://github.com/blue-build/.github/assets/60004820/7cfc3c0e-83e0-44bc-b9e2-16e5020ee595)

### Option B – Inspired by the [blue-billed curassow](https://duckduckgo.com/?q=blue-billed+curassow&iar=images&iax=images&ia=images)
Designed by @xynydev (me).

![Reference](https://github.com/blue-build/.github/assets/60004820/e97f05c3-b776-4048-a5a3-c4ecb7b0f6f9)

**Comment by tulilirockz** [comment]
Maybe the curassow one would be the best, Id just like to see more of the "Build" part of the name in the logo. Like, when I see that bird I think "hmmm cool blue bird..." but what about building software? Idk what could be done tho

**Reply by xynydev** [reply]
You can submit ideas and such. I'm not sure what I would do to add more of the build-ing, at least not something that would make the logo too generic. IMO a cool bird is a great logo, and I love the pun.

**Reply by gerblesh** [reply]
Tried to make a logo with a sort of "wrench-bird" lol, not at all what I would call "final" but it's an idea
![bluebuild-proto-logo](https://github.com/blue-build/.github/assets/101901964/2ff8bdd2-3dfd-4c0f-a789-e37a7740cfc8)
![bluebuild-proto-exported](https://github.com/blue-build/.github/assets/101901964/7f6852b7-c6c1-4a37-85bb-e9807c1cc0be)
(SVG font doesn't load properly)

**Reply by xynydev** [reply]
It looks kind of weire to me with the bird growing from the wrench IMO lol. Also the color hierarchy seems off with the dark-on-light-on-dark design. Seems that you used my logo as a base for this, I can see what I can do to explore this idea further.

For redoing a vote with more options I'd prolly have to go for http://star.vote/ so that the one picked everyone is alright with.

**Comment by fiftydinar** [comment]
Here's my logo proposal:

SVG:
![BlueBuild_logo+text](https://github.com/blue-build/.github/assets/65243233/367984ab-b222-494a-b715-4eb57b26ac0b)

PNG which will 100% load properly:
![BlueBuild_logo+text](https://github.com/blue-build/.github/assets/65243233/6ac8943d-2e05-4fa0-805c-bc279d8b2862)

Why I think it's good:
1. It uses wrench as a symbol for tinkering, which BlueBuild is intended for.
2. It also uses sun as a symbol of light, to indicate that tinkering can lead you to various improvements.
3. It matches closely Universal Blue colors, to pay the respect to OG (also matches Fedora colors).
4. I differentiate the simple & overused icon by using inner + outer shadows at different settings & by using more unusual icon shape. This makes logo have a bit of life in it & makes it more unique compared to most minimalist logos.
5. I use white accent only to the icon, to make the logo look consistent. This is the practice that Nokia used in their designs back then, which I consider great.
6. Thanks to icon shape, we can also fit slogan too below BlueBuild. I didn't think of which slogans yet.

What can be (maybe) improved:
1. xyny thinks that multitool is a more suitable icon compared to wrench. I agree, as BlueBuild offers much more then wrench. But multitool is usually a more complicated icon, which is harder to fit but to look good at the same time, so I think I cannot make it look better than the current wrench. Wrench indicates tinkering anyway, so I think it's still an okay fit.

Wrench logo uses Creative Commons 0 license:
https://www.svgrepo.com/svg/20710/wrench
https://www.svgrepo.com/page/licensing/#CC0

If you want to see previous designs of this logo, check my edits on this post.

Let me know thoughts on this y'all

**Reply by xynydev** [reply]
> Thanks to icon shape, we can also fit slogan too below BlueBuild. I didn't think of which slogans yet.

Doesn't fit on logo IMO.

> xyny thinks that multitool is a more suitable icon compared to wrench

That was kind of a half-joke, I'm not actually proposing a multitool logo. Paper.js already has one fun one http://paperjs.org/about/.

**Reply by fiftydinar** [reply]
> > Thanks to icon shape, we can also fit slogan too below BlueBuild. I didn't think of which slogans yet.
> 
> Doesn't fit on logo IMO.
> 
> > xyny thinks that multitool is a more suitable icon compared to wrench
> 
> That was kind of a half-joke, I'm not actually proposing a multitool logo. Paper.js already has one fun one http://paperjs.org/about/.

1. In most situations, yeah. I just remembered that Bazzite used it, so I noticed that it can be used in this logo too.
2. Thank God :P

I updated SVG with WOFF2 font instead of TTF.
I also added PNG picture, so anyone can view it 100% same as I intended.
So be sure to check it out again.

**Reply by fiftydinar** [reply]
I updated logo to use squircle-like icon shape as it fits better for the type of font used: "Atkinson Hyperlegible".
I tried regular circle & squircle fits better imo.

**Comment by xynydev** [comment]
Ok, I took the wrench & wrench-bird ideas and did my own takes on them with the same visual language as my previous bird-only logo. I quite like the wrench-bird I came up with lol. I quite like the colors and fonts I have, and I've started out with the design process for the website with the colors, so that's why I prefer my logos (well ofc I do lol...)

![image](https://github.com/blue-build/.github/assets/60004820/a8eac96f-c719-4b35-a1db-39ea5b3cda1b)

**Reply by xynydev** [reply]
Wrench icons used come from Googles set of Material symbols & icons
> We have made these icons available for you to incorporate into your products under the [Apache License Version 2.0](https://www.apache.org/licenses/LICENSE-2.0.txt). Feel free to remix and re-share these icons and documentation in your products. We'd love attribution in your app's about screen, but it's not required.

**Reply by fiftydinar** [reply]
If I would have to choose, I would select last icon without fill & with shine.

Website won't have blue-like colours to resemble UBlue/Fedora legacy though?

**Reply by xynydev** [reply]
> Website won't have blue-like colours to resemble UBlue/Fedora legacy though?

I planned that it would? 

> If I would have to choose, I would select last icon without fill & with shine.

That's coincidentally the one I like the least lol...

**Reply by fiftydinar** [reply]
> > Website won't have blue-like colours to resemble UBlue/Fedora legacy though?
> 
> I planned that it would?
> 
> > If I would have to choose, I would select last icon without fill & with shine.
> 
> That's coincidentally the one I like the least lol...

I probably made a mistake in judging these screenshots as how colors on the website would look like. I think it's better to experience website fully before I give conclusions

**Reply by xynydev** [reply]
Here's a previous design sketch for the website landing page hero. Nothing in this has been really made or finalized yet.

![image](https://github.com/blue-build/.github/assets/60004820/b263e7eb-8859-44cf-ad09-926db13d084d)

**Reply by fiftydinar** [reply]
My logo doesn't fit here at all.

If website design is going to be similar to this, then I think that minimalist Google-like logo would fit better. 1st three icons would fit nicely

**Reply by gerblesh** [reply]
I personally like the one in the top right, it's super easy to understand. Idk how we would include a bluebilled bird and have it "make sense" to someone viewing the logo (and keeping it minimal)

**Reply by xynydev** [reply]
> and have it "make sense" to someone viewing the logo

Logos don't make sense IMO, unless they're just boring wordmarks. They build their brand, their own symbol for their brand. A too generic logo is counterproductive IMO.

**Reply by xynydev** [reply]
> My logo doesn't fit here at all.

Of course, that's because I started designing something based on my logo for fun. I'd have to redo the design part if a logo using a different visual style gets picked. 

Worst part is, I want there to be _at least a dark mode_ (but probably dark&light), so if someone makes a light mode logo, it might look bad on a dark background...

**Reply by gmpinder** [reply]
I'm liking the webpage layout. Matches the simplicity goals of the project.

**Comment by xynydev** [comment]
## Get your submissions ready for the big logo vote!

Once everyone has submitted their logo proposals, I will create a poll on https://star.vote/ (most likely during the next three days). This will allow you to rate all logo proposals as you see fit, and the winner will be calculated such that it has the most approval. The victor will be examined for potential for improvement and I'll discuss with it's designer on how to fit it into it's different use cases.

### Guidelines

For each of your proposals (multiple allowed), include a single non-transparent image with your final logo design. If you want to get fancy, supply a reference image similar to the ones at the top of this discussion. Please also supply a short, neutral description of your logo, again similar to the ones up top ("Inspired by the blue-billed duck"). Logos that you don't feel are completely ready yet are allowed too, to prevent good ideas with hasty execution slipping through the cracks. To signify this, please add something like "status: unpolished" or "status: basically ready" to your description.

Submit your proposals in the replies to this comment.

**Reply by xynydev** [reply]
Inspired by the blue-billed duck. Status: basically ready. (Ian Price)

![image](https://github.com/blue-build/.github/assets/60004820/07730ba1-88f0-4769-a2b8-39c8586f0d44)

**Reply by xynydev** [reply]
Inspired by the blue-billed curassow. Status: basically ready.

![image](https://github.com/blue-build/.github/assets/60004820/b651cefb-0513-4b14-a8b5-771dc6f957af)

**Reply by xynydev** [reply]
Bird-wrench logo with by the blue-billed curassow. Status: room for improvement.

![image](https://github.com/blue-build/.github/assets/60004820/79d40b9d-0bc8-403b-9386-9f02c667fc8c)

**Reply by xynydev** [reply]
Spark-wrench with a dark blue theme. Status: room for improvement.

![image](https://github.com/blue-build/.github/assets/60004820/a29e280d-f013-40c0-b62a-a84cb2c84417)

**Reply by xynydev** [reply]
Duck hugging wrench. Status: very WIP. 

![image](https://github.com/blue-build/.github/assets/60004820/576b901f-07e9-41d8-9e0e-ab0fc05a2fd6)

**Reply by gerblesh** [reply]
Here's my slightly more cursed wrench-bird
![bluebuild-proto-exported](https://github.com/blue-build/.github/assets/101901964/f8f09a06-6f8f-44ec-89c1-21da3e8e9000)

**Reply by fiftydinar** [reply]
The reason why I did not include black theme is because dark theme is better for the eye strain in long term due to more balanced contrast.

Status: Ready

![BlueBuild_logo+text](https://github.com/blue-build/.github/assets/65243233/935d9592-dd46-4247-9d8d-ae29f1337682)
![BlueBuild_logo+text_dark2](https://github.com/blue-build/.github/assets/65243233/0027e334-1af4-48ad-adfc-efe2b0aaa438)

**Reply by xynydev** [reply]
Ducky. Status: basically ready.

![image](https://github.com/blue-build/.github/assets/60004820/c709cdef-4d78-4a8a-99e7-e5bacff52d69)

**Reply by fiftydinar** [reply]
BlueBuild font color looks too bright imo. Why not try to use that light blue accent color to it?

Otherwise, looks good

**Reply by xynydev** [reply]
Accent colored text looks like a link IMO. That color's the foreground/text color in the current palette, which I find it works well for.

**Reply by gmpinder** [reply]
I love it!

**Comment by xynydev** [comment]
## [VOTE](https://star.vote/bluebuildlogo/)

I created the vote now. Go to https://star.vote/bluebuildlogo/ to rate the logo proposals, winner will be determined based on that using the STAR-voting system. You can browse the logos above, or manually copy the image links to your address bar, since this site doesn't support images. I just thought that this type of voting is more just and gives people a better idea of how people like their logo. Sorry not sorry.

_I wonder which one will win... :duck:_

Edit: Anddd I appear to have copied in my own typo, "Bird-wrench logo **with by** the blue-billed curassow.", so much for professionalism...

**Reply by gmpinder** [reply]
Voted!

**Reply by fiftydinar** [reply]
Voted

**Reply by xynydev** [reply]
Ducky's a smash hit! I think we can call this now, doubt there's much more votes coming in the near vicinity.

**Reply by fiftydinar** [reply]
🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆🦆
